odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -2392,7 +2393,13 @@ def test_properties_field_read_group_date_week(self, date_type='date'):
         self.assertEqual(result[3]['attributes.mydate_count'], 1)
         self.assertEqual(result[3]['attributes.mydate:week'], 'W5 2023')
         self.assertEqual(result[4]['attributes.mydate_count'], 2)
-        self.assertEqual(result[4]['attributes.mydate:week'], 'W1 2022')
+        # Babel issue mitigation
+        # https://github.com/python-babel/babel/pull/621 -- introduced a new bug
+        # https://github.com/python-babel/babel/pull/887 -- proposed a fix but finally closed
+        # https://sources.debian.org/patches/python-babel/2.10.3-1/ -- Debian reverted 621
+        # so this ugly fix is made to have the test working in patched and non patched versions of Babel
+        expected = babel.dates.format_date(datetime.datetime(2023, 1, 1), "'W'w YYYY", "en_US")  # non patched: 'W1 2022' patched: 'W1 2023'
+        self.assertEqual(result[4]['attributes.mydate:week'], expected)
```

Comment:
I would do something more explicit 

```suggestion
        YEAR = babel.dates.format_date(datetime.datetime(2023, 1, 1), "YYYY", "en_US")
        if YEAR == '2022': # Broken babel, not really the desired behaviour
             self.assertEqual(result[4]['attributes.mydate:week'], 'W1 2022')
        else:  # Fixed babel
             self.assertEqual(result[4]['attributes.mydate:week'], 'W1 2023')
```